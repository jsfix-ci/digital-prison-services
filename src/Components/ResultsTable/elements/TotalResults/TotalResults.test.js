import React from 'react'
import { shallow } from 'enzyme'
import TotalResults from '.'

describe('<TotalResults />', () => {
  it('should match the default snapshot', () => {
    const wrapper = shallow(<TotalResults />)

    expect(wrapper).toMatchSnapshot()
  })

  it('should match with value snapshot', () => {
    const wrapper = shallow(<TotalResults totalResults={10} />)

    expect(wrapper).toMatchSnapshot()
  })
})
