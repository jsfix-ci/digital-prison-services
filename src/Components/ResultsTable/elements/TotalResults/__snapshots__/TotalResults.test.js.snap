// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TotalResults /> should match the default snapshot 1`] = `
<styled.div>
  Prisoners listed:
   
  <styled.span>
    0
  </styled.span>
</styled.div>
`;

exports[`<TotalResults /> should match with value snapshot 1`] = `
<styled.div>
  Prisoners listed:
   
  <styled.span>
    10
  </styled.span>
</styled.div>
`;
